packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -368,11 +368,14 @@ contract OptimismPortal2_FinalizeWithdrawal_Test is CommonTest {
     }
 
     /// @dev Tests that the guardian role can blacklist any dispute game.
-    function testFuzz_blacklist_guardian_succeeds(address _addr) external {
+    function testFuzz_blacklist_guardian_succeeds(IDisputeGame _addr) external {
+        vm.expectEmit(true, false, false, false, address(optimismPortal2));
```

Comment:
always using all true is a nice hedge in case the topics change, so new topics are automatically tested
```suggestion
        vm.expectEmit(address(optimismPortal2));
```